include/solvers/heat_transfer_assemblers.h

```diff
@@ -230,6 +230,45 @@ class HeatTransferAssemblerViscousDissipation
                StabilizedMethodsCopyData &   copy_data) override;
 };
 
+/**
+ * @brief Class that assembles the viscous dissipation for the heat transfer solver,
+ * for the specific case of VOF simulations. The only difference compared to the
+ * regular one is that the viscous dissipation can be applied in one of the
+ * fluids rather than both, through the viscous_dissipation_indicator parameter.
+ *
+ * @tparam dim An integer that denotes the number of spatial dimensions
+ *
+ * @ingroup assemblers
+ */
+template <int dim>
+class HeatTransferAssemblerViscousDissipationVOF
+  : public HeatTransferAssemblerBase<dim>
+{
+public:
+  HeatTransferAssemblerViscousDissipationVOF(
+    std::shared_ptr<SimulationControl> simulation_control,
+    Parameters::DissipationIndicator   p_viscous_dissipation_indicator)
+    : HeatTransferAssemblerBase<dim>(simulation_control)
+    , viscous_dissipation_indicator(p_viscous_dissipation_indicator)
+  {}
+
+  virtual void
+  assemble_matrix(HeatTransferScratchData<dim> &scratch_data,
+                  StabilizedMethodsCopyData &   copy_data) override;
+
+  /**
+   * @brief assemble_rhs Assembles the rhs
+   * @param scratch_data (see base class)
+   * @param copy_data (see base class)
+   */
+  virtual void
+  assemble_rhs(HeatTransferScratchData<dim> &scratch_data,
+               StabilizedMethodsCopyData &   copy_data) override;
+
+
+protected:
+  Parameters::DissipationIndicator viscous_dissipation_indicator;
```

Comment:
I think it would be clearer if the parameter had the same name as the variable, but it is really up to you. I feel like if someone else is modifying it, the person may want to do a ctrl+f and find everything about it.

Reply:
Yes you are right! It was a previous name of the parameter, I will change that